Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
>I think we can probably handle this on the brew side of things, but, I'm not sure it's going to be pretty :confused:.

No, because post install overwrites it and we synchronized the versions. You're saying the versions can no longer be synchronized, which means those will need to be changed to conflicts_with instead of keg_only, unless I've missed something.